tests/apollo/util/bft.py

```diff
@@ -165,6 +166,8 @@ def start_replica_cmd(builddir, replica_id, config)
     third parameter named 'config' (the exact name is important!).
     If you don't need this configuration - use two parameters callback with any names you want.
     """
+    assert not (break_on_first_failure and break_on_first_success), \
```

Comment:
You are checking this condition twice, remove this redundant check and keep the one in repeat_test

Reply:
I prefer doubl checking, since I have no idea how th decorator will be used in the future.
If we have another with_bft_network1 function, and it is not checking in the decorator, we might get bugs.
In general, double checking in 2 different places is never a bad thing when therei. good reason for it.

Reply:
This is exactly the point, it doesn't matter how the decorator (repeat_test) is used, because it validates its inputs.
if you implement with_bft_network__1 and use the repeat_test decorator, you'll be validated, and if you won't use it, then check for yourself like you did.